airbyte-scheduler/src/main/java/io/airbyte/scheduler/temporal/SpecWorkflow.java

```diff
@@ -91,46 +89,29 @@ public SpecActivityImpl(ProcessBuilderFactory pbf, Path workspaceRoot) {
       this.workspaceRoot = workspaceRoot;
     }
 
-    public ConnectorSpecification run(IntegrationLauncherConfig launcherConfig) {
+    public JobOutput run(IntegrationLauncherConfig launcherConfig) {
       try {
-        // todo (cgardens) - we need to find a way to standardize log paths sanely across all workflow.
-        // right now we have this in temporal workflow.
+        // todo (cgardens) - there are 2 sources of truth for job path. we need to reduce this down to one,
+        // once we are fully on temporal.
         final Path jobRoot = workspaceRoot
-            .resolve("spec")
-            .resolve(launcherConfig.getDockerImage().replaceAll("[^A-Za-z0-9]", ""))
-            .resolve(String.valueOf(Instant.now().getEpochSecond()));
+            .resolve(String.valueOf(launcherConfig.getJobId()))
+            .resolve(String.valueOf(launcherConfig.getAttemptId().intValue()));
+
+        MDC.put("job_id", String.valueOf(launcherConfig.getJobId()));
```

Comment:
Are you planning to DRY the MDC config?

Reply:
yeah. jared just did it in his PR. i'm hoping we can do a base class of some sort so that we don't need to have to remember to do these over and over again.